src/test/rustdoc/tuple_struct_fields.rs

```diff
@@ -0,0 +1,11 @@
+// @has tuple_struct_fields/struct.Tooople.html
```

Comment:
Could you also add a check like:

```suggestion
// @has tuple_struct_fields/struct.Tooople.html
// @has - 'Tuple Fields'
```

to make sure the heading is correct?

Reply:
I think i added that at one point then removed it because it wasn't working, but ill try again tomorrow 👍

Reply:
> (and the errors are less helpful than staring at a wall)

Yeah, htmldocck errors could use some work.